docs/user/installation.rst

```diff
@@ -125,13 +125,16 @@ Alternatively, it can be done manually:
 2. mkdir ~/python-pycurl-openssl
 3. cd ~/python-pycurl-openssl
 4. sudo apt-get source python-pycurl
-5. sudo apt-get build-dep python-pycurl
-6. sudo apt-get install libcurl4-openssl-dev
-7. sudo dpkg-source -x pycurl_7.19.0-3build1.dsc
-8. cd pycurl-7.19.0
-9. edit debian/control file and replace all instances of “libcurl4-gnutls-dev” with “libcurl4-openssl-dev”
-10. sudo PYCURL_SSL_LIBRARY=openssl dpkg-buildpackage -rfakeroot -b
-11. sudo dpkg -i ../python-pycurl_7.19.0-3build1_i386.deb
+5. sudo apt-get build-dep python-pycurl -y
+6. sudo apt-get install libcurl4-openssl-dev -y
+*** CAUTION: BE CAREFUL WITH THIS OR DELETE THE DIRECTORY MANUALLY TO BE SAFE ***
+7. sudo rm -r ./*/ ; dpkg-source -x pycurl_7*.dsc # *** CAUTION: BE CAREFUL WITH THIS OR DELETE THE DIRECTORY MANUALLY TO BE SAFE ***
```

Comment:
Do you need the rm or Can you change it to something more specific?

Reply:
This is the shortest path; I wrote it as a simple workaround in order to work with future versions. It has to be either deleted or the source has to be extracted to a custom path in order to avoid dpkg-source error complaining about existing target:
dpkg-source: error: unpack target exists: pycurl-7.43.0.2
 
2 alternatives to the potentially unsafe "rm -r ./*/" :

1) find . ! -path . -type d -prune -exec rm -r {} \;

2) Another option would be to extract it to a custom directory like this:
dpkg-source -x pycurl_7*.dsc <output directory>